pyproject.toml

```diff
@@ -1,10 +1,186 @@
+[project]
+name = 'asdf'
+description = 'Python implementation of the ASDF Standard'
+readme = 'README.rst'
+requires-python = '>=3.7'
+license = { file = 'LICENSE' }
+authors = [{ name = 'The ASDF Developers', email = 'help@stsci.edu' }]
+classifiers = [
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.7',
```

Comment:
We also don't technically support python 3.7

Reply:
should the following lines be changed in `master` then?
https://github.com/asdf-format/asdf/blob/80c3a84a00fd96708ce485f2d4c05088bf7f6116/setup.cfg#L18
https://github.com/asdf-format/asdf/blob/80c3a84a00fd96708ce485f2d4c05088bf7f6116/setup.cfg#L26